generator/internal/rust/templates/crate/src/transport.rs.mustache

```diff
@@ -68,17 +68,13 @@ impl super::stub::{{Codec.Name}} for {{Codec.Name}} {
         req: {{InputType.Codec.QualifiedName}},
         options: gax::options::RequestOptions,
     ) -> Result<gax::response::Response<{{Codec.ReturnType}}>> {
-        let options = gax::options::internal::set_default_idempotency(
-            options,
-            {{PathInfo.Codec.IsIdempotent}},
```

Comment:
😢

Reply:
Option A: return the precomputed `(builder, default_idempotency, body)`

Option B: return `(builder, http_method)` then compute the default idempotency and the NoBody in some cases

Which do you prefer? I am happy to do either

Reply:
Option C: leave as-is and we can fix this later.

Reply:
Sounds like B -> A.

Ack, thanks for the opinion.